libs/s25main/world/GameWorldBase.h

```diff
@@ -59,6 +60,7 @@ class GameWorldBase : public World
     EventManager& em;
     std::unique_ptr<SoundManager> soundManager;
     LuaInterfaceGame* lua;
+    std::unique_ptr<Cheats> cheats;
```

Comment:
don't know if this needs to be an unique_ptr? can't you simply call the ctor in GameWorldBase ctor?

Reply:
No reason other than to forward-declare instead of including. This can just be `Cheats cheats;` if this is preferred. I didn't want to recompile so much code when working on the Cheats header but I can see the benefits of removing the unique_ptr crap (clarity, unnecessary to include Cheats.h in some other places).

Reply:
I'll leave the unique_ptr. Including Cheats.h from GameWorldBase.h is just a pain and several other members of GameWorldBase are also unique_ptr's (presumably for a similar reason). Please unresolve if you disagree.

Reply:
Ah yes, it's another cyclic dependency here. Moving it out of the world should resolve it.